src/Submissions/Api/Areas/Biobank/Controllers/ProfileController.cs

```diff
@@ -93,7 +93,8 @@ public class ProfileController : Controller
         _publicationsConfig = publicationsOptions.Value;
         _userManager = userManager;
         _signInManager = signInManager;
-    }
+    _logoService = logoService;
```

Comment:
```suggestion
        _logoService = logoService;
```
(indentation consistent)